index.html

```diff
@@ -917,8 +1072,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_SEPARATOR</code> and object attribute
 											<code>xml-roles:doc-pagebreak</code>. </p></td>
-								<td>AXRole: <code>AXSplitter</code><br /> AXSubrole: <code>&lt;nil&gt;</code><br />
-									AXRoleDescription: <code>'splitter'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXSplitter</code></li>
+										<li>AXSubrole: <code>&lt;nil&gt;</code></li>
```

Comment:
TBD… A new subrole may be useful for page breaks, but since these aren't UI page breaks, but print media page breaks, the API would probably not align.

Reply:
Confirming as-is. Use splitter role with additional "page break" description.

Reply:
Yes I would like to see the additional "page break" description, as there may be other splitters which are not due to page breaks.